manual/src/cmds/intf-c.etex

```diff
@@ -2403,66 +2410,67 @@ section.
 \subsection{ss:c-thread-register}{Registering threads created from C}
 
 Callbacks from C to OCaml are possible only if the calling thread is
-known to the OCaml run-time system.  Threads created from OCaml (through
+known to the OCaml runtime system.  Threads created from OCaml (through
```

Comment:
It would be _much_ easier to review PRs like this if stylistic changes like this could be in a separate PR, or at least in a separate commit. `git add -p` exists for a reason!

You've missed one on L847

Reply:
I am familiar with the `--patch` command line and I frequently use it to split my commits in units of sense when appropriate. I think the present commit was simple enough.

> You've missed one on L847

In _"computed at run-time"_, we have an adjective formed from two words separated with a dash. This is proper English, unlike _"computed at runtime"_.

Note that while the change is stylistic (_"run-time system"_ is perfectly correct English), most of the documentation already uses "runtime system" instead.